dspy/evaluate/evaluate.py

```diff
@@ -108,8 +109,8 @@ def __call__(
                 `self.num_threads`.
             display_progress (bool): Whether to display progress during evaluation. if not provided, use
                 `self.display_progress`.
-            display_table (bool): Whether to display the evaluation results in a table. if not provided, use
-                `self.display_table`.
+            display_table (Tuple[bool, int]): Whether to display the evaluation results in a table. if not provided, use
+                `self.display_table`. If a number is passed, the evaluation results will be truncated to that number.
```

Comment:
Should just be the number (max rows)

Reply:
If we'll drop the support for bool here, let's show a warning for one cycle and change the behavior?